vmm/src/lib.rs

```diff
@@ -1248,18 +1134,17 @@ impl Vmm {
     }
 
     fn start_microvm(&mut self) -> std::result::Result<VmmData, VmmActionError> {
-        START_INSTANCE_REQUEST_CPU_TS.store(now_cputime_us() as usize, Ordering::Release);
-        START_INSTANCE_REQUEST_TS.store(
-            (chrono::Utc::now().timestamp_nanos() / 1000) as usize,
-            Ordering::Release,
-        );
         info!("VMM received instance start command");
         if self.is_instance_initialized() {
             return Err(VmmActionError::StartMicrovm(
                 ErrorKind::User,
                 StartMicrovmError::MicroVMAlreadyRunning,
             ));
         }
+        let request_ts = TimestampUs {
+            time_us: (chrono::Utc::now().timestamp_nanos() / 1000) as u64,
```

Comment:
nit: maybe add an inline shorthand for `(chrono::Utc::now().timestamp_nanos() / 1000) as u64`, like `now_cputime_us`.

Reply:
This is real time not cpu time, but I added an inline function for it.